base/setup/usetup/usetup.c

```diff
@@ -626,6 +626,171 @@ UpdateKBLayout(VOID)
     }
 }
 
+/*
+ * Function Name: ProgressTimeOutStringHandler
+ *
+ * Purpose: Handles the generation (displaying) of the timeout countdown to the screen
+ *          dinamically
+ *
+ * Return: FALSE
+ */
```

Comment:
Header is not compatible with https://reactos.org/wiki/Coding_Style
Somebody should start following our own code-style :)